main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"cloudfoundry.org/cf-k8s-api/apis"
+	"cloudfoundry.org/cf-k8s-api/routes"
+	"fmt"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+	"os"
+
+	. "cloudfoundry.org/cf-k8s-api/config"
+)
+const defaultConfigPath = "config.json"
+var GlobalConfig Config
```

Comment:
GlobalConfig is exported, but then only ever used inside the body of the main func. I'd suggest deleting it and switching the LoadConfigFromPath() func to return a config instead of taking in a pointer to mutate

Reply:
See [ec3de58a915baa698d7c229c2dccb67f4a9b8c05](https://github.com/cloudfoundry/cf-k8s-api/pull/5/commits/ec3de58a915baa698d7c229c2dccb67f4a9b8c05)